jenkins-x-boot-vault-tls.yml

```diff
@@ -0,0 +1,59 @@
+buildPack: none
+pipelineConfig:
+  pipelines:
+    pullRequest:
+      pipeline:
+        options:
+          containerOptions:
+            resources:
+              limits:
+                cpu: 4
+                memory: 6144Mi
+              requests:
+                cpu: 1
+                memory: 2048Mi
+        environment:
+          - name: DOMAIN_ROTATION
+            value: "true"
+          - name: DOMAIN_PREFIX
+            value: "boot"
+          - name: DOMAIN_SUFFIX
+            value: ".bdd.jenkins-x.rocks"
+          - name: GOPROXY
+            value: http://jenkins-x-athens-proxy:80
+          - name: JX_REQUIREMENT_VAULT_DISABLE_URL_DISCOVERY
+            value: "true"
+          - name: GKE_SA
+            value: /secrets/bdd/sa.json
+          - name: JX_REQUIREMENT_VAULT_DISABLE_URL_DISCOVERY
+            value: "true"
+          - name: GH_ACCESS_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: jenkins-x-bot-test-github
+                key: password
+          - name: JENKINS_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: test-jenkins-user 
+                key: password
+        agent:
+          image: gcr.io/jenkinsxio/builder-go-maven2.0.897-247
```

Comment:
ah-ha, here's the problem - should be `gcr.io/jenkinxsio/builder-go-maven2:2.0.897-247` - the `:` is missing.